cmd/edit/machinepool/cmd_test.go

```diff
@@ -0,0 +1,33 @@
+package machinepool
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+)
+
+// Will un-comment and become the cmd test once the work on refactoring this command is finalized
+var _ = Describe("Edit Machinepool", func() {
```

Comment:
Moved all tests in here to `pkg/machinepool/machinepool_test.go` -- except the one commented out here as it tests the actual command, which will be rewritten in the next MR immediately following this one.